iati/tests/test_validator.py

```diff
@@ -525,6 +540,20 @@ def test_xml_check_valid_xml_detailed_output(self, xml_str, error_log_empty):
 
         assert result == error_log_empty
 
+    def test_xml_check_explicit_encoding_in_str_detailed_output(self, xml_str_explicit_encoding, error_log_empty):
+        """Perform check to see whether a parameter is valid XML.
+
+        The parameter is valid XML, but in a format that lxml does not support.
+        Obtain detailed error output.
+        """
+        result = iati.validator.validate_is_xml(xml_str_explicit_encoding)
+
+        if sys.version_info.major > 2:
```

Comment:
Should add a compatibility note =)

Reply:
495b0c0035ed438bd909781982e1c964e0f8228e